### DISCUSSION TITLE
Is HTML meta refresh supported?

### DISCUSSION THREAD

**Discussion by rewiaca** [projectdiscovery/nuclei#1254] [Q&A] [answered]
Does nuclei scanner support this type of redirect?
```
<html>
<head>
<meta http-equiv="refresh" content="0; URL=https://google.com">
</head>
<body></body></html>
```
I suppose headless is definitely supports this but, any way to handle it more efficiently? 

`redirects` and `max-redirects` options are can't handle it, is it only manual parsing and making new request?

**Comment by ehsandeep** [comment, marked as answer]
@rewiaca `redirects` and `max-redirects` works with the default engine of nuclei using standard go **net/http** to make HTTP request which doesn't follow javascript-based redirection and are only supported with headless templates using a headless browser.
